README.rst

```diff
@@ -10,3 +10,18 @@ The preferred way is that you put a link to your own repository here:
 - https://github.com/mrkmg/borgbackup-zsh-completion (zsh completion for borg)
 - https://github.com/obilodeau/borgbackup-scripts (cron script with xdg desktop notifications)
 - (add your link above this line)
+
+Debugging tools
+---------------
+
+- https://gist.github.com/enkore/14f7bd9f56d6cc17914a73345fd30fc4
+
+  Shows the most commonly referenced cache entries; useful to diagnose corrupted caches.
+  
+- https://gist.github.com/enkore/92216e919bc0ad2602d4a429bed597fb
+
+  Shows freeable space in a repository, mostly interesting for 1.1.x repos and --append-only repos.
```

Comment:
this could / should be part of the "borg compact" command that was considered some time ago